src/utils/validateNric.tsx

```diff
@@ -1,55 +1,7 @@
 // Thanks to https://gist.github.com/kyrene-chew/6f275325335ab27895beb7a9a7b4c1cb
```

Comment:
nit: since we are removing the validation, perhaps we can remove the acknowledgement too!